### PR TITLE
updating background image code for homepage

### PR DESCRIPTION
**Pull request by NicoleMarty**

### CODE REVIEW

File: views/index.handlebars [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div id="home" style="background:url('/images/cocktails.jpg'); background-size: cover;">
+<body style="background:url('/images/cocktails.jpg'); background-size: cover;">
     <div class="container-fluid">
```
Comment (btgoligowski24) [inline views/index.handlebars]: remove the inline style tag for preference of external stylesheet

File: views/index.handlebars [hunk 2]
```diff
@@ -18,16 +18,16 @@
...
-        crossorigin="anonymous"></script>
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="/js/index.js"></script>
-</div>
+</body>
+<!-- BOOTSTRAP JS -->
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+    integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+    crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
```
Comment (btgoligowski24) [inline views/index.handlebars]: was this the code provided to us or was it some that we found? i ran into an issue with minified code in a previous homework and it drove me freaking nuts until i learned what the issue was. i believe it was that `$.ajax` was removed from minified code. so we should keep an eye out for errors.

Comment (NicoleMarty) [reply views/index.handlebars]: I usually use minified just to save space, but good to know. I can always change it to.

Comment (NicoleMarty) [reply views/index.handlebars]: meh I'll just change it now ha

Comment (NicoleMarty) [reply views/index.handlebars]: actually.... jk. I only see the minified version on the bootstrap website...

Comment (btgoligowski24) [reply views/index.handlebars]: yeah, they only have the smaller version. i usually just google jquery cdn and the first result is one.

Comment (btgoligowski24) [reply views/index.handlebars]: <script
  src="https://code.jquery.com/jquery-3.4.1.js"
  integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
  crossorigin="anonymous"></script>

Comment (NicoleMarty) [reply views/index.handlebars]: ok got it in there. thanks!

**Pull request closed without merging**